src/Transition.js

```diff
@@ -212,7 +212,7 @@ class Transition extends React.Component {
     if (timeout != null && typeof timeout !== 'number') {
       exit = timeout.exit
       enter = timeout.enter
-      appear = timeout.appear
+      appear = timeout.appear !== undefined ? timeout.appear : enter
```

Comment:
let's add a TODO here to remove this fallback on the next major bump

Reply:
added